content/scout/integrations/environment/sysdig.md

```diff
@@ -0,0 +1,90 @@
+real-time insights about your security posture, and lets you compare your
+builds with what's running in production.
+
+## How it works
+
+The Sysdig Agent captures the images of your container workloads. Docker Scout
+integrates with the Sysdig API to discover the images in your cluster.
+
+Each Sysdig integration maps to an environment. When you establish a Sysdig
+integration, you specify the environment name for that cluster, such as
+`production` or `staging`. Docker Scout assigns the images in the cluster to
+the corresponding environment. This lets you use the environment filters to see
+vulnerability status and policy compliance for an environment.
+
+The prerequisites for integrating Docker Scout with Sysdig are:
+
+- You must install the Sysdig Agent in the cluster that you want to integrate.
+  For more information, see [Sysdig docs](https://docs.sysdig.com/en/docs/installation/).
+- You must enable the runtime insights feature in Sysdig.
+
+The images that you deploy must already be known by Docker Scout through
+[registry integrations](../_index.md#container-registries). The Sysdig runtime
+integration only assigns known images to environments. It doesn't analyze
+images.
+
+## Integrate an environment
+
+You must be an organization owner to enable this integration.
+
+1. Go to the [Sysdig
+   integrations](https://scout.docker.com/settings/integrations/ecr) page on
+   the Docker Scout Dashboard.
+
+2. Go to [Integrations](https://scout.docker.com/settings/integrations/) on the
+   Docker Scout Dashboard.
+3. Select the **Detect images running in my environments** filter.
+4. Find Sysdig in the list, and select **Integrate**.
+
+5. In the **How to integrate** section, enter a configuration name and a Risk
+   Spotlight API token.
+
+   Configuration name is the display name of the Sysdig integration in Docker
+   Scout. The Risk Spotlight API token is the Sysdig token that Docker Scout
+   needs to integrate with Sysdig. For more information about Risk Spotlight
+   Integrations in Sysdig, [see here](https://docs.sysdig.com/en/docs/sysdig-secure/integrations-for-sysdig-secure/risk-spotlight-integrations/).
+
+6. Under **Configuration name**, enter a display name for this integration.
+7. In the second input field, add the Risk Spotlight Integration API token.
+8. Select **Next**.
+9. Enter the cluster name.
+
+   This corresponds to the `global.clusterConfig.name` configuration parameter
+   you set when deploying the Sysdig Agent.
+
+10. Select **Next**.
+11. Select the region in the drop-down list.
+
+    This corresponds to the `global.sysdig.region` configuration parameter you
+    set when deploying the Sysdig Agent.
+
+12. Select **Next**.
+
+    After selecting **Next**, Docker Scout performs a connection test for the
+    configuration you've specified. An error displays if the connection test
+    fails. In the case of an error, go back and make sure you've entered the
+    configuration details correctly.
```

Comment:
**issue:** the user won't be able to advance if the connection fails, they'll be stuck on the previous step